src/configs.rs

```diff
@@ -31,12 +39,20 @@ lazy_static! {
         AtomicI64::new(BLOOM_MEMORY_LIMIT_PER_FILTER_DEFAULT);
     pub static ref BLOOM_USE_RANDOM_SEED: AtomicBool = AtomicBool::default();
     pub static ref BLOOM_DEFRAG: AtomicBool = AtomicBool::new(BLOOM_DEFRAG_DEAFULT);
+    pub static ref BLOOM_FP_RATE_F64: Mutex<f64> =
+        Mutex::new(BLOOM_FP_RATE_DEFAULT.parse::<f64>().unwrap());
```

Comment:
Let's use expect instead of unwrap
```suggestion
        Mutex::new(BLOOM_FP_RATE_DEFAULT.parse::<f64>().expect("Expected valid f64 for fp rate."));
```